src/controllers/commands/CommandController.cpp

```diff
@@ -181,28 +181,6 @@ bool appendWhisperMessageWordsLocally(const QStringList &words)
     return true;
 }
 
-bool appendWhisperMessageStringLocally(const QString &textNoEmoji)
```

Comment:
I'm not too confident in removing this, but I couldn't really see a reason for this to exist other than maybe some `dryRun` stuff.